### ISSUE TITLE
ETC Altcoin not appearing in Wallet view (of the Accounts tab in Parity Client)

### ISSUE THREAD

**Issue by ltfschoen** [paritytech/parity#7306]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Which Parity version?**: v1.8.4-beta
* Note: same issue occurs with Parity v1.8.3-beta
> - **Which operating system?**: MacOS
* macOS Sierra 10.12.6
> - **How installed?**: Installer
* Downloaded https://parity-downloads-mirror.parity.io/v1.8.4/x86_64-apple-darwin/parity-1.8.4-macos-installer.pkg
* Validated checksum with `shasum -a 256 parity-1.8.4-macos-installer.pkg`
* Run parity-1.8.4-macos-installer.pkg
* Updating to latest Google Chrome Version 63.0.3239.108 (Official Build) (64-bit)
> - **Are you fully synchronized?**: yes
> - **Did you try to restart the node?**: yes
* Yes, I've restarted Parity, restarted Google Chrome, and emptied browser cache with hard reload

_Your issue description goes here below. Try to include **actual** vs. **expected behaviour** and **steps to reproduce** the issue._

**Actual Behaviour** 
The Wallet view in the Account tab of the Parity Client does not show any Ethereum Classic (ETC) holdings. Even though it is a different chain, is this a feature that the community thinks would look favourably upon including? When the owner of that Ethereum address goes to check their ETC balance at http://gastracker.io/addr/<YOUR_ETHEREUM_ADDRESS> it shows the ETC balance for an Ethereum address that has money on both the ETH and ETC chains. I assume it would be convenient to see those holdings near the section that has the tokens associated with an Ethereum address in the Parity Client, but perhaps clearly highlighted as being part of another chain.

**Expected Behaviour** 
If your Ethereum address has a balance on both the ETH and ETC chains, whether as a result of the ETH/ETC split in 2016, or as a result of accidentally sending ETC to an ETH address, then in your Wallet view (of the Accounts tab in the Parity Client) it could also show the ETC balance corresponding to that Ethereum address under a heading like "Other Altcoins", just like it shows ERC20 tokens that correspond to that Ethereum address. Otherwise the owner of that Ethereum address may not be aware that they hold an ETC balance

**Steps to Reproduce**
* Option 1: Send ETC to an ETH address then check your ETC account balance for that address at this link http://gastracker.io/addr/<YOUR_ETHEREUM_ADDRESS>. **Warning** Recovering the ETC by separating your tokens on each chain after performing this action is risky and may be susceptible to a replay attack 
* Option 2: Find an Ethereum address that had a balance before the ETH/ETC hard fork of 2016. Check its ETC account balance at this link http://gastracker.io/addr/<YOUR_ETHEREUM_ADDRESS>

**Comment by 5chdn** [comment]
There is no way to know the balance without synchronizing the chain.

For ETH run `parity --chain foundation`

For ETC run `parity --chain classic`

There is currently no plan to enable synchronization of two chains in parallel.
